_distributor_init.py

```diff
@@ -22,5 +22,12 @@
     libs_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                              '.libs'))
     if os.path.isdir(libs_path):
+        # for Python < 3.8, try to achieve brute force resolution
+        # of DLLs by moving into their storage dir; should have no
+        # effect for Python >= 3.8
```

Comment:
why "should have no effect"? being more explicit would be good here. and/or skip the `chdir` by checking `sys.version`

Reply:
also adding a rationale or link to the relevant issue would be useful

Reply:
The gruesome details are now in a substantially expanded set of comments above the code block.  Note that we never really understood the 32-bit vs. 64-bit Python 3.7 issue, but simply spread out our DLL resolution mechanisms to include "one old way" and "one new way," and that seemed to work.

You could certainly make the argument that the `chdir` stuff should be restricted to Python < 3.8, since that works and is now the most robust/secure way to load DLLs in std lib. I haven't done that, but we could if you prefer.